package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kiltprotocol/sdk-js",
-  "version": "0.19.0",
+  "version": "0.19.1-3",
```

Comment:
this will probably remain bc these prereleases are already published, right?

Reply:
yes